src/main/java/cn/nukkit/block/BlockDriedKelp.java

```diff
@@ -0,0 +1,16 @@
+package cn.nukkit.block;
+
+import cn.nukkit.utils.BlockColor;
+import cn.nukkit.utils.Identifier;
+
+public class BlockDriedKelp extends BlockSolid {
```

Comment:
Formatting in this class is completely wrong

Reply:
I've auto-formatted the file. Can you review again?